.eslintignore

```diff
@@ -0,0 +1,9 @@
+.build
+lib
+.cache
+.idea
+__pycache__
+logs/
+chromium-bidi.iml
+node_modules/
+chromium-bidi-*
```

Comment:
What is `chromium-bidi-*` ?

NIT: no new line

Reply:
tar.gz files produced by `npm pack`